src/app/components/change-request/change-requests-wrapper.jsx

```diff
@@ -1,27 +1,38 @@
 import React from 'react';
 import { arrayOf, string } from 'prop-types';
+import { makeStyles } from '@material-ui/core';
 
 import ChplChangeRequests from './change-requests';
 
 import AppWrapper from 'app-wrapper';
 import BreadcrumbWrapper from 'components/breadcrumb/breadcrumb-wrapper';
 
+const useStyles = makeStyles(() => ({
```

Comment:
```suggestion
const useStyles = makeStyles({
```

The "function" format is used when you need to pass something into the object. We don't need that in almost all of our pages, and therefor is unnecessary extra code. Likely will need to remove this stuff in a few more places